srcpkgs/stw/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'stw'
+pkgname=stw
+version=0.2
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
```

Comment:
```suggestion
make_use_env=yes
```

Reply:
this wont change anything but i guess i'll add it.

Reply:
> feel free to open a PR to use yes in them (one commit per package, no revbumps).

i have no idea how to do that and git is scary, so..
also yeah thank you for pointing that out.